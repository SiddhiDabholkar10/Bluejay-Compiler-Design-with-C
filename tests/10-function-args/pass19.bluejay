implicit samename int := 400;

fun f int ()
  var samename int := 473;
  return samename;


return f();
