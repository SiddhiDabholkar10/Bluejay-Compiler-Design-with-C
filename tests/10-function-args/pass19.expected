Function declared "f" returns int
	Local variable "samename" type int position 0
Argument/local read "samename" type int frame position 0
Function called "f" returns int
